app/experimenter/experiments/tests/test_forms.py

```diff
@@ -202,6 +205,320 @@ class Meta:
         self.assertEqual(change.old_status, old_status)
         self.assertEqual(change.new_status, new_status)
 
+    def test_mixin_changelog_values(self):
+        experiment = Experiment()
+
+        data = {
+            "type": Experiment.TYPE_ADDON,
+            "status": Experiment.STATUS_ACCEPTED,
+            "name": "This is an experiment!",
+            "short_description": "my short description",
+            "related_work": "some other experiment",
+            "risk_release_population": True,
+            "review_impacted_teams": True,
+        }
+
+        class TinyTestForm(ChangeLogMixin, forms.ModelForm):
+
+            class Meta:
+                model = Experiment
+                fields = (
+                    "type",
+                    "status",
+                    "name",
+                    "short_description",
+                    "related_work",
+                    "risk_release_population",
+                    "review_impacted_teams",
+                )
+
+        form = TinyTestForm(
+            request=self.request, data=data, instance=experiment
+        )
+
+        self.assertTrue(form.is_valid())
+        form.save()
+        self.assertEqual(experiment.changes.count(), 1)
+        self.assertEqual(experiment.changes.latest().new_values, data)
+
+    def test_mixin_changelog_values_with_prev_log(self):
```

Comment:
@jaredkerim  just want to make sure i understood correctly, you want me to scrap this and keep 
https://github.com/mozilla/experimenter/pull/1361/files#diff-3d5bec13bbcacb975a33ef9256fa8a4eR302?